src/tests/integration/test-data/show/Identity.applyOp(uint256).cse.expected

```diff
@@ -55,7 +55,7 @@
 
 
 
-module SUMMARY-SRC%CSE%IDENTITY.APPLYOP(UINT256):0
+module SUMMARY-SRC%CSE%IDENTITY.APPLYOP(UINT256):1
```

Comment:
This change is a bit worrying, why is there a change in version?

Reply:
This is fixed now. I think it came from me trying to run/update dependencies on multiple test groups with one command.